test/gui/shared/scripts/pageObjects/PublicLinkDialog.py

```diff
@@ -246,3 +266,13 @@ def verifyResource(self, resource):
             str(squish.waitForObjectExists(self.ITEM_TO_SHARE).text),
             resource,
         )
+
+    def verifyExpirationDate(self, expectedDate):
+        expectedDate = datetime.strptime(expectedDate, '%Y-%m-%d')
+        expYear = expectedDate.year - 2000
```

Comment:
Oh! I didn't realize that the year is composed of only 2 digits. May be add some comment here?

Reply:
I didn't get you?

Reply:
I meant, add a comment here, describing the logic.